packages/jest-mock/src/index.ts

```diff
@@ -969,11 +973,8 @@ export class ModuleMocker {
       metadata.value = component;
       return metadata;
     } else if (type === 'function') {
-      // @ts-expect-error this is a function so it has a name
```

Comment:
oh, nice!

Reply:
I was also happy, but that’s a false positive. Types got `any`ed. Turning them back to `unknown`